src/test/spec/unified_runner/operation.rs

```diff
@@ -343,39 +309,65 @@ impl TestOperation for DeleteOne {
 }
 
 #[derive(Debug, Default, Deserialize)]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
Removing `deny_unknown_fields` is unfortunate but this was already in an unsupported situation by using it with a `flatten`ed field, and outright broke when it was itself used as such.

Reply:
technically it's a requirement of the unified test format that we error on unrecognized fields, though it would be quite a bit of boilerplate to adhere to that here, so not sure if it's worth it or not.

Reply:
I'd personally go with "not" - I think not having to deal with keeping multiple duplicate data structures in sync is the lesser evil here - but I don't know what precedent is like.

Reply:
Updated.